test/testing_utils.dart

```diff
@@ -0,0 +1,224 @@
+import 'angular_utils.dart';
+import 'dart:math';
+import 'package:webdriver/io.dart';
+import 'package:random_string/random_string.dart' as random;
+import 'dart:async';
+import 'dart:io';
+
+void startSelenium() {
+  print("Starting Selenium Chrome Driver");
+  Process.run(
+      'java.exe -Ddriver.chrome.driver=chromedriver.exe -jar '
+      'selenium-server-standalone-3.6.0.jar',
+      []);
+}
+
+Future<WebDriver> setupWebDriver(int width, int height) async {
+  var options = {'browserName': 'chrome'};
+  WebDriver driver = await createDriver(desired: options);
+  driver.logs
+      .get('browser')
+      .where((LogEntry l) => l.level == 'SEVERE')
+      .listen((LogEntry l) => print(l));
+  await (await driver.window).setSize(new Rectangle<int>(0, 0, width, height));
+  return driver;
+}
+
+Future navigateToPage(WebDriver driver, String slug) async {
+  await driver.get("https://www.gyerunkanyukam.hu/" + slug);
+}
+
+Future<String> signUpNewRandomUser(WebDriver driver, String pw) async {
+  var newRandomUserEmail;
+  var loginBtn = await driver
+      .findElement(const By.cssSelector('div.menu-item.menu-item--login > a'));
+  await loginBtn.click();
+  var signUpBtn = await driver
+      .findElement(const By.xpath('//*//tab-button[2]/material-ripple'));
+  await signUpBtn.click();
+  var signUpEmail = await driver.findElement(const By.xpath(
+      '*//label[contains(@class,"input-container")]/input[contains(@type,"email")]'));
+  newRandomUserEmail = generateRandomEmail();
+  await signUpEmail.sendKeys(newRandomUserEmail);
+  var signUpPW = await driver.findElement(const By.xpath(
+      '*//label[contains(@class,"input-container")]/input[contains(@type,"password")]'));
+  await signUpPW.sendKeys(pw);
+  var termsRadioBtn = await driver.findElement(const By.xpath(
+      "*//material-checkbox/div[contains(@class, 'icon-container')]/material-ripple"));
+  await termsRadioBtn.click();
+  var submitBtn =
+      await driver.findElement(const By.xpath('*//input[@type="submit"]'));
+  await submitBtn.click();
+  await waitForAngular(driver);
+  await sleep5();
+  return newRandomUserEmail;
+}
+
+String generateRandomEmail() {
+  return random.randomAlphaNumeric(10) + "@email.com";
+}
+
+Future loginUserWithCredentials(
```

Comment:
This should be moved to the other test groups (it is not a generic purpose utility function).